agent/lib/kontena/agent.rb

```diff
@@ -82,6 +82,10 @@ def supervise_launchers
     end
 
     def supervise_network_adapter
+      @supervisor.pool(
```

Comment:
I understood from Celluloid docs that you shouldn't really do that: https://github.com/celluloid/celluloid/wiki/Pools#gotcha-dont-make-pools-inside-workers